configure.ac

```diff
@@ -168,6 +168,7 @@ CC_CHECK_FLAGS_APPEND([with_cflags], [CFLAGS], [\
         -Werror=implicit-function-declaration \
         -Werror=missing-declarations \
         -Werror=return-type \
+        -Werror=format2 \
```

Comment:
OOI, where is that documented? https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html describes `format=2` but not `format2`?